### PR TITLE
Debian: fix HostonlyInitrd and autologin

### PR DESCRIPTION
**Pull request by bluca**

### CODE REVIEW

File: mkosi/__init__.py [hunk 1]
```diff
@@ -1432,6 +1432,7 @@ def configure_dracut(args: CommandLineArguments, root: str) -> None:
     ):
         with open(os.path.join(dracut_dir, "30-mkosi-uefi-stub.conf"), "w") as f:
             f.write("uefi_stub=/usr/lib/systemd/boot/efi/linuxx64.efi.stub\n")
+            f.write("add_drivers+=efivarfs\n")
```
Comment (DaanDeMeyer) [inline mkosi/__init__.py:1435]: The other config files enclose dracut lists in quotes with spaces. If possible, this should do that as well.

File: mkosi/__init__.py [hunk 2]
```diff
@@ -2883,7 +2887,7 @@ def set_autologin(args: CommandLineArguments, root: str, do_run_build_script: bo
 
     os.chmod(override_file, 0o644)
 
-    pam_add_autologin(root, "pts/0")
+    pam_add_autologin(root, device_prefix + "pts/0")
```
Comment (DaanDeMeyer) [inline mkosi/__init__.py:2890]: Nit, we prefer python f-strings (`f"{device_prefix}pts/0"). Ditto for the other changes.

File: mkosi/__init__.py (hunk 1 above)
Comment (bluca) [reply mkosi/__init__.py]: fixed

File: mkosi/__init__.py (hunk 2 above)
Comment (bluca) [reply mkosi/__init__.py]: fixed

**Review by DaanDeMeyer — APPROVED** [review]
LGTM, let's wait for CI to finish before merging

**Pull request merged**